solarforecastarbiter/datamodel.py

```diff
@@ -820,6 +822,7 @@ class Report(BaseModel):
     end: pd.Timestamp
     forecast_observations: Tuple[ForecastObservation, ...]
     metrics: Tuple[str, ...] = ('mae', 'mbe', 'rmse')
+    categories: Tuple[str, ...] = ('total', 'date', 'hour')
```

Comment:
I don't think it necessarily has to be all datetime, but I don't necessarily see any that wouldn't be.  Perhaps we may want to separate grouping by magnitude or into ramp up or ramp down?  Perhaps something more descriptive would be `report_categories`?

Reply:
At this point I'm ok keeping it as a general `categories` and allowing for the possibility of non-datetime options. All of the attributes could be described as `report_something`, so probably best to avoid `report_` prefix.

Reply:
Forgot to ask what the defaults, if any, should be for this attribute. I'm ok with no defaults or `('total', )`.

Reply:
Total is fine with me. Updated